oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoConnectionTest.java

```diff
@@ -85,26 +95,6 @@ public void sufficientReadConcern() throws Exception {
         sufficientReadConcernSingleNode(ReadConcern.MAJORITY, true);
     }
 
-    @Test
-    public void socketKeepAlive() throws Exception {
```

Comment:
Again, I would keep this alive till we remove the config.

Reply:
As before, Mongo doesn't have any related `isSocketKeepAlive` method anymore. The only way to keep this test is to make it a NOOP test, which doesn't make much sense to me.

Reply:
In [Mongo Java Driver 3.6](https://mongodb.github.io/mongo-java-driver/3.6/javadoc/?com/mongodb/MongoClientOptions.html) it was already deprecated, and in 4.0 removed.